packages/smithy-core/src/smithy_core/__init__.py

```diff
@@ -8,6 +8,8 @@
 from . import interfaces, rfc3986
 from .exceptions import SmithyException
 
+__version__: str = "0.1.0"
```

Comment:
Same note here, are we bumping the version? I would think we do our first minor on the initial release.

Reply:
Ditto on pyproject version here: https://github.com/smithy-lang/smithy-python/blob/develop/packages/smithy-core/pyproject.toml#L3